meta-lmp-bsp/dynamic-layers/freescale-layer/recipes-bsp/firmware-imx/firmware-imx_8.%.bbappend

```diff
@@ -0,0 +1,4 @@
+do_install:append() {
```

Comment:
if this is needed here, I wonder if this is needed upstream.
I'm up to reworking the firmware-imx_8 recipe mainline. I'm adding a note here, so that when I'm working on this, I remember where it's coming from